### ISSUE TITLE
Ruby 2.7 Deprecation Warning

### ISSUE THREAD

**Issue by oniofchaos** [scoutapp/scout_apm_ruby#319]
```
/app/vendor/bundle/ruby/2.7.0/gems/scout_apm-2.6.6/lib/scout_apm/instruments/active_record.rb:333: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

**Comment by ioquatix** [comment]
@cschneid would you like me to sort this out?

**Comment by dlanderson** [comment]
Thanks! This was addressed in https://github.com/scoutapp/scout_apm_ruby/pull/354 and released in 4.0.0 - Closing
